src/components/spreadsheet/table-wrapper.js

```diff
@@ -562,7 +562,7 @@ const TableWrapper = (props) => {
             const wrappedEditedData = {
                 data: editingData,
             };
-            Object.entries(priorValuesBuffer).forEach(([field, value]) => {
```

Comment:
Even if a field isn't used, I would let it for others developers knowing there is others fields available.

Reply:
I let it back, thank you